src/main/java/com/powsybl/openloadflow/ac/nr/CustomNewtonRaphsonStoppingCriteria.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
```

Comment:
The copyright header should have the licence identifier "SPDX-License-Identifier: MPL-2.0" as the last line.